src/blockchain/transactions.ts

```diff
@@ -202,15 +203,13 @@ export function send(
   const result: Observable<TxState> = bindNodeCallback(method)(...args).pipe(
     mergeMap((txHash: string) => {
       const broadcastedAt = new Date();
-      return timer(0, 1000).pipe(
+      return interval(500).pipe(
         switchMap(() => bindNodeCallback(web3.eth.getTransaction as GetTransaction)(txHash)),
-        takeWhileInclusive(transaction => !transaction),
+        tap(console.log),
```

Comment:
Is it necessary? If it is, it should produce a meaningfull message.

Reply:
of course it's not :). I will remove it right away